modules/rapid/include/opencv2/rapid.hpp

```diff
@@ -0,0 +1,124 @@
+
+implements "RAPID-a video rate object tracker" @cite harris1990rapid
+*/
+
+namespace cv
+{
+namespace rapid
+{
+//! @addtogroup rapid
+//! @{
+
+/**
+ * debug draw markers of matched correspondences onto a lineBundle
+ * @param img the lineBundle
+ * @param srcLocations the according source locations
+ * @param newLocations matched source locations
+ * @param colors colors for the markers. Defaults to white.
+ */
+CV_EXPORTS_W void drawCorrespondencies(InputOutputArray img, InputArray srcLocations,
+                                       InputArray newLocations, InputArray colors = noArray());
+/**
+ * debug draw search lines onto a video frame
+ * @param img the video frame
+ * @param locations the source locations of a line bundle
+ * @param color the line color
+ */
+CV_EXPORTS_W void drawSearchLines(InputOutputArray img, InputArray locations, const Scalar& color);
+
+/**
+ * draw a wireframe of a triangle mesh
+ * @param img the output image
+ * @param pts2d the 2d points obtained by @ref projectPoints
+ * @param tris triangle face connectivity
+ * @param color line color
+ * @param type line type. See @ref LineTypes.
+ * @param cullBackface enable back-face culling based on CCW order
+ */
+CV_EXPORTS_W void drawWireframe(InputOutputArray img, InputArray pts2d, InputArray tris,
+                                const Scalar& color, int type = LINE_8, bool cullBackface = false);
+/**
+ * extract control points from the projected silhoutte of a mesh
+ * @param num number of control points
+ * @param len search line radius (used to restrict the ROI)
+ * @param pts3d the 3D points of the mesh
+ * @param rvec rotation between mesh and camera
+ * @param tvec translation between mesh and camera
+ * @param K camera intrinsic
+ * @param imsize size of the video frame
+ * @param tris triangle face connectivity
+ * @param ctl2d the 2D locations of the control points
+ * @param ctl3d matching 3D points of the mesh
+ */
+CV_EXPORTS_W void extractControlPoints(int num, int len, InputArray pts3d, InputArray rvec, InputArray tvec,
+                                       InputArray K, const Size& imsize, InputArray tris, OutputArray ctl2d,
+                                       OutputArray ctl3d);
+/**
+ * extract the line bundle from a video frame
+ * @param len the search line radius. The bundle will have `2*len + 1` columns.
+ * @param ctl2d the search lines will be centered at this points and orthogonal to the contour defined by
+ * them. The bundle will have as many rows.
+ * @param frame the image to read the pixel intensities values from
+ * @param bundle the resulting bundle
+ * @param srcLocations the source locations in the image for each bundle location
+ */
+CV_EXPORTS_W void extractLineBundle(int len, InputArray ctl2d, InputArray frame, OutputArray bundle,
+                                    OutputArray srcLocations);
+
+/**
+ * Find corresponding image locations by searching for a maximal sobel edge along the search line (a single
```

Comment:
If a sentence is the first sentence in the line, it starts with a lower-case letter.

(My preferred solution would of course be to stick to English and start all sentences with an upper-case letter and end all sentences with a period. That would also look much nicer in the generated documentation :) )